docs/benchmarks/data-caching.md

```diff
@@ -73,43 +58,47 @@ The client can simultaneously use multiple servers, one server with several ip a
     IP_ADDRESS_VM2, 11211
     ...
 
-You can use the `vim` command for modifying this file inside the container.
+
+
+To start the client container use the following command:
+
+    $ docker run -idt --name dc-client --net host -v PATH_TO_DOCKER_SERVERS_FOLDER:/usr/src/memcached/memcached_client/docker_servers/ cloudsuite/data-caching:client
+
+Please note that the command mounts the folder containing the 'docker_servers.txt' file instead of mounting only the file. This way, further changes to the docker_servers.txt file in the host will be reflected inside of the container. 
 
 #### Scaling the dataset and warming up the server ####
 
 The following command will create the dataset by scaling up the Twitter dataset, while preserving both the popularity and object size distributions. The original dataset consumes 300MB of server memory, while the recommended scaled dataset requires around 10GB of main memory dedicated to the Memcached server (scaling factor of 30).
 
-    $ ./loader -a ../twitter_dataset/twitter_dataset_unscaled -o ../twitter_dataset/twitter_dataset_30x -s docker_servers.txt -w 4 -S 30 -D 4096 -j -T 1
-
-(`w` - number of client threads which has to be divisible to the number of servers, `S` - scaling factor, `D` - target server memory, `T` - statistics interval, `s` - server configuration file, `j` - an indicator that the server should be warmed up).
+    $ docker exec -it dc-client /bin/bash /entrypoint.sh --m="S&W" --S=30 --D=1024 --w=8 --T=1
```

Comment:
1024 to 10240

Reply:
Do we need `exec /bin/bash /entrypoint.sh`, I though by default if we pick `run`, it will do the equivalent to that command?

Reply:
I think we need it because we are using the `exec` command and this command needs a binary to run. I am not using `run` command because we have broken the client's tasks into multiple subtasks, and we call each separately using the `exec` command.

Reply:
@Xusine1131 Actually the dataset with scale factor 30 is bigger than 10 GB. So maybe we recommend 11 GB instead of 10 GB in the document.

Reply:
The scaling factor is a function of the recommended dataset size, which should lead to a representative state for measurement.